### ISSUE TITLE
error in plugin-window-events

### ISSUE THREAD

**Issue by phoet** [appsignal/appsignal-javascript#452]
`null is not an object (evaluating 'e.reason.stack')`

Mac OSX, Apple Safari 14.0

```
    function _onUnhandledRejectionHandler(error) {
      var span = self.createSpan();
      span.setError({
        name: "UnhandledPromiseRejectionError",
        message: typeof error.reason === "string" ? error.reason : JSON.stringify(error.reason),
        stack: error.reason.stack || ""
      });
      self.send(span);
```

if you want, take a look at it yourself: https://appsignal.com/onruby/sites/5dec086c747820155d483106/exceptions/incidents/257?timestamp=2020-12-03T08%3A11%3A14Z

**Comment by xadamy** [comment]
Thanks @phoet! ✌️ I've patched this in #453, and I will release a new version as soon as I can.

**Comment by phoet** [comment]
😽  thx!
